app/Http/Controllers/TrainingDash.php

```diff
@@ -802,6 +777,17 @@ public function saveNewTrainerFeedback(Request $request) {
         $redirect = ($request->input('redirect_to') == 'internal') ? '/dashboard' : '/';
         return redirect($redirect)->with('success', 'Thank you for the feedback! It has been received successfully.');
     }
+
+    public function handleSchedule() {
+        $user = Auth::user();
+
+        if (!$user->onboarding_complete) {
```

Comment:
Please make this logic check only applicable to users with a rating of OBS.

Reply:
Resolved and squashed into 5d41d2b